oak_containers_launcher/src/server.rs

```diff
@@ -45,9 +53,23 @@ impl Launcher for LauncherServerImplementation {
         &self,
         _request: Request<()>,
     ) -> Result<Response<Self::GetOakSystemImageStream>, tonic::Status> {
+        let system_image_file = tokio::fs::File::open(&self.system_image).await?;
+
+        let mut buffer = [0; MAX_RESPONSE_SIZE];
+        let mut reader = BufReader::new(system_image_file);
+
         let response_stream = async_stream::try_stream! {
-          // TODO(#4023): Send a system image
-          yield GetImageResponse::default();
+            loop {
+                let bytes_read = reader.read(&mut buffer).await?;
+
+                if bytes_read > 0 {
+                    yield GetImageResponse {
+                        image_chunk: buffer.to_vec()
+                    }
+                } else {
+                    break;
+                }
+            }
```

Comment:
I believe this could be written more nicely using tokio's `FramedRead` but this implementation was for now. Might revisit in the future to make it nicer.